.github/test_on_push.yml

```diff
@@ -0,0 +1,53 @@
+name: pybamm_cookiecutter
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
Should we also run it with `push: [main]`?

Reply:
added it in the latest commit